Dockerfile.builder

```diff
@@ -21,25 +20,45 @@ RUN set -ex \
   done
 
 #ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 6.5.0
+ENV NODE_VERSION 6.9.2
```

Comment:
@ldimaggi this fix didn't make it through the rebasing. Please change this back to 6.5.0.